Description: Fix FTBFS with Clang instead of GCC
Bug-Debian: http://bugs.debian.org/753266
Applied-Upstream: 2.4.0
Last-Update: 2016-02-04

--- a/eiskaltdcpp-qt/src/HubFrame.cpp
+++ b/eiskaltdcpp-qt/src/HubFrame.cpp
@@ -772,7 +772,7 @@
     }
 }
 
-HubFrame::HubFrame(QWidget *parent=NULL, QString hub="", QString encoding=""):
+HubFrame::HubFrame(QWidget *parent, QString hub, QString encoding):
         QWidget(parent),
         d_ptr(new HubFramePrivate())
 {
--- a/eiskaltdcpp-qt/src/HubFrame.h
+++ b/eiskaltdcpp-qt/src/HubFrame.h
@@ -128,7 +128,7 @@
     typedef QMap<QString, QVariant> VarMap;
     typedef QList<ShellCommandRunner*> ShellList;
 
-    HubFrame(QWidget *parent, QString, QString);
+    HubFrame(QWidget *parent = NULL, QString hub = "", QString encoding = "");
     ~HubFrame();
 
     bool parseForCmd(QString, QWidget *);
